app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, source_file, target_file = command.split()
+    if cmd != "cp" or source_file == target_file:
+        return
+    try:
+        with (open(source_file, "r") as source,
+              open(target_file, "w") as target):
+            target.write(source.read())
```

Comment:
This line can be inefficient for large files, as it loads the entire content of the source file into memory. Consider using a loop to read the source file and write to the target file in chunks.